database/query/profile_status.sql

```diff
@@ -125,10 +126,10 @@ SELECT
     ad.alert_details,
     ad.alert_metadata,
     ad.alert_last_updated,
-    res.id AS rule_evaluation_id,
-    res.repository_id,
-    res.entity,
-    res.rule_name,
+    ed.id AS rule_evaluation_id,
+    ere.repository_id,
+    ere.entity_type,
+    ri.name AS rule_name,
     repo.repo_name,
     repo.repo_owner,
     repo.provider,
```

Comment:
I agree that pulling this repo stuff out here is a bit weird (especially since it seems GitHub-specific!).  How much does it improve things if we drop these contents from the RPC?  (And, is anyone using them right now / when were they added?)

Reply:
If we _do_ need them, we could do a second query in the RPC, which feels like a better place, to be honest.

Reply:
I think we should get rid of them, but removing them will impact the UI (from what I understand, one of the endpoints which uses this query powers the "Alerts" page in the UI) so we need to co-ordinate this change.

Reply:
What do you think of collecting the repo and other entity details in a _separate_ database call outside of this query?

If the frontend is using that information (and it makes sense that they would), it seems a shame to make them make two RPCs instead of one.  At the same time, I don't want this query getting bigger if we can avoid it, so maybe doing some iteration in the Minder server is better than extending the query.